infra/gcp/ensure-staging-storage.sh

```diff
@@ -189,6 +189,9 @@ function ensure_staging_gcs_bucket() {
 
     color 6 "Ensuring ${writers} can write to ${bucket} in project: ${project}"
     empower_group_to_write_gcs_bucket "${writers}" "${bucket}"
+
+    # Ensure logging is turned on
+    ensure_gcs_bucket_logging ${bucket}
```

Comment:
This ends up calling `ensure_gcs_bucket_logging gs://foo`

Reply:
Of course, yes. ty